libs/wingsdk/src/helpers.ts

```diff
@@ -24,3 +37,7 @@ export function range(start: number, end: number, inclusive: boolean) {
   }
   return iterator();
 }
+
+export function nodeof(construct: Construct): Node {
+  return Node.of(construct);
```

Comment:
Does this work to avoid extra guff bundled when inflights use helpers?

```suggestion
  const Node = require("./std/node").Node;
  return Node.of(construct);
```

Reply:
Just tried it out, it looks like the bundle size is the same both ways